static/app/views/eventsV2/table/tableView.tsx

```diff
@@ -182,12 +183,27 @@ class TableView extends Component<TableViewProps & WithRouterProps> {
         isHomepage,
       });
 
-      return [
+      const eventIdLink = (
         <Tooltip key={`eventlink${rowIndex}`} title={t('View Event')}>
           <StyledLink data-test-id="view-event" to={target}>
             {value}
           </StyledLink>
-        </Tooltip>,
+        </Tooltip>
+      );
+
+      return [
+        organization.features.includes('discover-quick-context') ? (
+          <QuickContextHoverWrapper
+            key={`quickContextEventHover${rowIndex}`}
+            dataRow={dataRow}
+            contextType={ContextType.EVENT}
+            organization={organization}
+          >
+            {eventIdLink}
+          </QuickContextHoverWrapper>
+        ) : (
+          eventIdLink
+        ),
       ];
```

Comment:
Also would flip this logic

```suggestion
      if (!organization.features.includes('discover-quick-context')) {
        return [eventIdLink];
      }
      
      return [
        <QuickContextHoverWrapper
          key={`quickContextEventHover${rowIndex}`}
          dataRow={dataRow}
          contextType={ContextType.EVENT}
          organization={organization}
        >
          {eventIdLink}
        </QuickContextHoverWrapper>
      ];
```